src/main/java/com/neuronrobotics/bowlerstudio/lipsync/RhubarbManager.java

```diff
@@ -95,12 +95,14 @@ public void processRaw(File f, String ttsLocation) throws Exception {
 		List<Map<String, Object>> cues = (List<Map<String, Object>>) resultParsed.get("mouthCues");
 		for (Map<String, Object> cue : cues) {
 			double end = Double.parseDouble(cue.get("end").toString());
-			double percent = end / duration * 100.0;
-
+			double start = Double.parseDouble(cue.get("start").toString());
+//			double percent = end / duration * 100.0;
+//
 			AudioStatus val = AudioStatus.get(cue.get("value").toString().charAt(0));
-			System.out.println("End at " + percent + " " + val);
-			HashMap<AudioStatus, Double> map = new HashMap<>();
-			map.put(val, percent);
+//			System.out.println("End at " + percent + " " + val);
+//			HashMap<AudioStatus, Double> map = new HashMap<>();
+//			map.put(val, percent);
+			TimeCodedViseme map = new TimeCodedViseme(val, start, end, duration);
```

Comment:
@madhephaestus i'm seeing obj "TimeCodedViseme" and ALSO "TimeCodedVisemes" ... do both of these object definitions exist?